### PR TITLE
Raise when forced alternative is not present in alternatives list

### PR DESCRIPTION
**Pull request by drnluz**

### CODE REVIEW

File: lib/sixpack.rb [hunk 1]
```diff
@@ -69,17 +69,20 @@ def participate(experiment_name, alternatives, force=nil, kpi=nil, traffic_fract
         raise ArgumentError, "Invalid traffic fraction, must be between 0 and 1" unless traffic_fraction.to_f.between?(0,1)
       end
 
+      if !alternatives.include?(force)
```
Comment (vsh91) [inline lib/sixpack.rb]: You have to check that force is not nil.

**Review by vsh91 — CHANGES_REQUESTED** [review]
Is this PR ready? Some specs are broken. Could you please add specs that assert that error raised?

**Comment by drnluz** [comment]
@vsh91 Added the spec, thanks for the review

**Review by vsh91 — APPROVED** [review]

**Comment by vsh91** [comment]
@drnluz thanks.

**Pull request merged**